subcommittees/mentoring/PracticumProposal.md

```diff
@@ -0,0 +1,192 @@
+history.  Expert help will be provided to support the mentors. 
+- Time commitment: 1 hr of training (discussed below), 1 hr online / week for 3-4 weeks of mentoring,
+2-3 hrs / week on email mentoring, ~2 hrs / group member on mini-lesson preparation, observation, and
+follow-up.  Mentoring for this practicum is a time-limited focused committment; volunteering now
+doesn’t mean you’re committing to doing it again 
+- Training, discussion questions, and feedback tools will be provided 
+  
+Mentors can apply via a form that requests the following information: 
+
+- SCF experience 
+- Other teaching experience 
+- R or python preference 
+- Time zone plus preferred time of day for meetings (daytime/evening) 
+
+Mentors will be selected and matched to groups by the mentoring committee.  
+
+Before starting to meet with their groups, all mentors are required to attend the mentoring training
+meeting. 
+
+### Lead mentor
+
+There will be a lead mentor who will lead the mentors in their work. That person will be in charge of
+mentor training, and for keeping track of the progress of the groups by staying in touch with the
+mentors. 
+
+### Mentoring
+
+Mentors will go through a one hour training meeting which will include an overview of responsibilities,
+materials for facilitating discussions with their groups, methods of giving and receiving feedback, and
+a review of the code of conduct.
+
+It is the mentor’s responsibility to email their group members and set up a time to meet and an
+expected lesson schedule (this can be over email or via an initial meeting).  
+
+Each week, the group meets without the mentor to discuss their experience with the lesson.  The group
+also meets with their mentor once a week for a more in-depth discussion.  Depending on the number of
+groups and mentors, one mentor may be responsible for two groups.  We suggest that students meet on
+Tues/Wed, with mentor on Friday or weekend, if convenient for the group. 
+
+### Mini-lessons
+
+The following is the proposed mini-lesson plan:  
+
+1. Trainees will in collaboration with their mentor figure out a portion of material that the student
+will teach from. The aim is to have approximately an hours worth of material. The trainee will then
+prepare the material, including data, and make it available for review by the mentor. 
+2. The mentor will then review the lesson plan and have a short (online) meeting with each trainee one
+week before the trainee is scheduled to teach their Mini-lesson. Guidance on what to teach and how to
+teach it will be given both via email and during face-to-face communication. 
+3. The trainee teaches the 1 hr Mini-lesson, in person, to a minimum of 1 other individual while the
+mentor observes either in person, or remotely. 
+4. Directly after the Mini-lesson, the trainee and mentor engage in a debrief where the mentor first
+asks the trainee to comment on their performance before they give feedback on the trainee’s performance. 
```

Comment:
I think the reasoning is that the student should get to speak first, before the mentor's opinions get heard. A trainee is likely to put a lot of emphasis on what the mentor would say, and could thus end up with ignoring their own opinions.